### DISCUSSION TITLE
希望首页的推荐视频类型可以选择性过滤

### DISCUSSION THREAD

**Discussion by veizei-K** [the1812/Bilibili-Evolved#4289] [功能建议 / Ideas]
![image](https://github.com/the1812/Bilibili-Evolved/assets/129288685/647ef289-e62d-42c4-b48a-8e543a1820bf)
比如这些电影直播和纪录片什么的

**Comment by Geeyun-JY3** [comment]
清爽首页下好像是支持的，根据动态过滤器里的关键词。

**Reply by the1812** [reply]
是的, 原版首页不会支持.
